src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -28,9 +29,12 @@ import {
   RouteRateModelUpdate,
   ConfigStoreVersionUpdate,
   DisabledChainsUpdate,
-} from "../interfaces";
+  UBAConfigUpdates,
+  UBAParsedConfigType,
+} from "../../interfaces";
```

Comment:
This sets off some alarm bells for me - does the ConfigStoreClient class need to know about UBA concepts and types? It seems like an inversion to me.

Reply:
These types are specifically defined within the `./interfaces/ConfigStore.ts` file. They represent the on-chain definition of the UBA Configuration (similar to the rate model). 

I think the ConfigStore will need to know about the uba concepts insofar as to parse out the config from our configstore contract

cc @nicholaspai

Reply:
On second review I can see that there are already other upper-layer primitives that we depend on in here - i.e. `L1TokenTransferThreshold` and `SpokePoolTargetBalance`.

It still generally seems circular to me, because the ConfigStoreClient otherwise sits at the very low layers of our part of the stack, but it ends up knowing about high level concepts. I may just have the wrong perspective on it though, and (in light of the above) the changes being introduced are at least consistent with the existing usage :shrug:.

Reply:
I think where you put the `UBAConfigUpdates` and `UBAParsedConfigType` type, in the `interfaces/ConfigStore` makes sense to me. The ConfigStoreClient definitely needs to know the shape of the UBA config in order to parse the on chain `uba` key into the correct curves